packages/tc-markdown-parser/lib/tree-mutators/create-treecreeper-name-node.js

```diff
@@ -0,0 +1,61 @@
+const { selectAll } = require('unist-util-select');
```

Comment:
maybe this would be better named as create-treecreeper-title-node.js, as this takes the title of the doc and writes it into a field (which maybe should default to code, actually, and runbook.md can configure it to write to name instead)